### PR TITLE
[09-hsinchu] Create sprint page

### PR DESCRIPTION
**Pull request by YenPeiChen07**

### CODE REVIEW

File: content/pages/sprint/2022/09-hsinchu.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+Cost sharing
+------------
+
+This occurrence is free.
+
+Venue
+-----
+
+`清華大學 綜合三館A區 5樓 <https://goo.gl/maps/HPmvSiVwvSjUkQ7dA>`__. Please see the `sprint
```
Comment (yungyuc) [inline content/pages/sprint/2022/09-hsinchu.rst:34]: The Google Map link leads to "General Physics Laboratory, NTHU 清華大學普物實驗室", which reads a little bit misleading.  How about using just the building's name, not a specific unit in it?

**Review by yungyuc — CHANGES_REQUESTED** [review]
Everything looks good except the Google Map link.

File: content/pages/sprint/2022/09-hsinchu.rst (hunk 1 above)
Comment (YenPeiChen07) [reply content/pages/sprint/2022/09-hsinchu.rst:34]: Yes, it seems a little bit misleading. I will modify it by using the building's name.

**Comment by YenPeiChen07** [comment]
@yungyuc I have updated the Google Map link of the venue. Also changed the building's name so that it fit with the Google Map link.

**Comment by yungyuc** [comment]
LGTM.  Thanks a lot!

**Pull request merged**